packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/Credentials.kt

```diff
@@ -52,5 +66,68 @@ public interface Credentials {
         public fun emailPassword(email: String, password: String): Credentials {
             return CredentialImpl(CredentialImpl.emailPassword(email, password))
         }
+
+        /**
+         * Creates credentials representing a login using a user API key.
```

Comment:
should we link/explain who provides/creates the given `key`?

Reply:
I have added a link to the docs.